builder/builder.go

```diff
@@ -1410,6 +1409,41 @@ func (b *Builder) createMixBundleDir() error {
 	return nil
 }
 
+// If Base == true, template will include the [main] and [clear] sections.
+// If Local == true, template will include the [local] section.
+type dnfConf struct {
+	UpstreamURL, RepoDir string
+	Base, Local          bool
+}
+
+const dnfConfTemplate = `{{if .Base}}[main]
+cachedir=/var/cache/yum/clear/
+keepcache=0
+debuglevel=2
+logfile=/var/log/yum.log
+exactarch=1
+obsoletes=1
+gpgcheck=0
+plugins=0
+installonly_limit=3
+reposdir=/root/mash
+
+[clear]
+name=Clear
+failovermethod=priority
+baseurl={{.UpstreamURL}}/releases/$releasever/clear/x86_64/os/
+enabled=1
+gpgcheck=0
+{{end}}{{if .Local}}
+[local]
+name=Local
+failovermethod=priority
+baseurl=file://{{.RepoDir}}
+enabled=1
+gpgcheck=0
+priority=1
+{{end}}`
+
 // BuildBundles will attempt to construct the bundles required by using the m4
```

Comment:
Outdated comment!

Reply:
Thank you. This also reminded me that `m4` also needed to be removed from the dependency list. Commit added to fix both.